packages/protocol/migrationsConfig.js

```diff
@@ -97,8 +97,12 @@ const DefaultConfig = {
     participationBaselineQuorumFactor: 1,
   },
   governanceApproverMultiSig: {
-    //Placeholder until addresses for 2/5 multsig are generated.
-    signatories: [`0x5409ed021d9299bf6814279a6a1411a7e866a631`],
+    // 1/3 placeholder until addresses for 2/5 multsig are generated.
```

Comment:
Can we make this 2/3 for both regular and rquired. And feel free to remove this comment

Reply:
Also FYI one more address will be ready in an hour, let's try to get that in this PR

Reply:
i'm guessing you mean for regular and internal?

so it will then be 2/4 when we have the new address

Reply:
Yeah 2/4 for both regular and internal for now